libs/labelbox/pyproject.toml

```diff
@@ -89,9 +93,10 @@ unit = "pytest tests/unit"
 # LABELBOX_TEST_BASE_URL="http://host.docker.internal:8080" \
 integration = { cmd = "pytest tests/integration" }
 data = { cmd = "pytest tests/data" }
+rye-lint = "rye lint"
 rye-fmt-check = "rye fmt --check"
 mypy-lint = "mypy src --pretty --show-error-codes --non-interactive --install-types"
-lint = { chain = ["mypy-lint", "rye-fmt-check"] }
+lint = { chain = ["rye-fmt-check", "mypy-lint", "rye-lint"] }
```

Comment:
do we still need mypy here?

Reply:
Yea I think so Mypy is a [type checker](https://docs.astral.sh/ruff/faq/#how-does-ruff-compare-to-mypy-or-pyright-or-pyre)